iteration.qmd

```diff
@@ -429,14 +431,12 @@ If needed, you could `pivot_wider()` this back to the original form.
     nycflights13::flights |> show_missing(c(year, month, day))
     ```
 
-## Reading multiple files
+## Ανάγνωση πολλαπλών αρχείων
 
-In the previous section, you learned how to use `dplyr::across()` to repeat a transformation on multiple columns.
-In this section, you'll learn how to use `purrr::map()` to do something to every file in a directory.
-Let's start with a little motivation: imagine you have a directory full of excel spreadsheets[^iteration-5] you want to read.
-You could do it with copy and paste:
-
-[^iteration-5]: If you instead had a directory of csv files with the same format, you can use the technique from @sec-readr-directory.
+Στην προηγούμενη ενότητα μάθατε πώς να χρησιμοποιείτε την `dplyr::across()` για να επαναλάβετε έναν μετασχηματισμό σε πολλαπλές στήλες.
+Σε αυτήν την ενότητα θα μάθετε πώς να χρησιμοποιείτε την `purrr::map()` για να εφαρμόσετε οτιδήποτε σε κάθε αρχείο ενός καταλόγου.
+Ας ξεκινήσουμε με ένα παράδειγμα: φανταστείτε ότι έχετε έναν φάκελο γεμάτο αρχεία Excel\[\^επανάληψη-5\] που θέλετε να διαβάσετε.
+Μπορείτε να το κάνετε με αντιγραφή και επικόλληση για κάθε ένα αρχείο:
```

Comment:
Λείπει η μετάφραση αυτού του κομματιού: 

[^iteration-5]: If you instead had a directory of csv files with the same format, you can use the technique from @sec-readr-directory.